test/index.js

```diff
@@ -9,6 +9,7 @@ import Z from 'sanctuary-type-classes';
 
 import doctest from '../lib/doctest.js';
 
+import {major} from './node-version.js';
```

Comment:
consider `import * as nodeVersion` so the code at the usage site(s) becomes more self-explanatory: `if (nodeVersion.major >= 14)`

Reply:
Good idea! Updated. :)